conda.recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: dask
+  # GIT_DESCRIBE_TAG is supposed to return the latest version (0.4.0) but it is
+  # not working for some reason. And PKG_VERSION is incorrect (it says 1.0.1k)
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '0.4.0')}}
```

Comment:
Oh, is it possible that you don't have jinja2 (or whatever templating system this is) installed locally?